projects/BFDR/NatalityRecord_submissionProperties.cs

```diff
@@ -6639,7 +6639,7 @@ public string MotherMarriedDuringPregnancyHelper
         public string PaternityAcknowledgementSignedHelper
         {
             get => GetObservationValueHelper();
-            set => SetObservationValueHelper(value, VR.ValueSets.YesNoNotApplicable.Codes);
+            set => SetObservationValueHelper(value, VR.ValueSets.YesNoUnknownNotApplicable.Codes);
```

Comment:
in the IG it looks like the original value set was correct, shouldn't it be Y,N,NA https://build.fhir.org/ig/HL7/fhir-bfdr/branches/master/StructureDefinition-Observation-paternity-acknowledgement-signed.html

Reply:
According to the IJE spreadsheet, it should be YesNoUnknownNotApplicable. Creating a ticket on the IG side to make this fix.